x-pack/plugins/siem/public/containers/helpers.ts

```diff
@@ -12,4 +12,4 @@ import { ESQuery } from '../../common/typed_json';
 export const createFilter = (filterQuery: ESQuery | string | undefined) =>
   isString(filterQuery) ? filterQuery : JSON.stringify(filterQuery);
 
-export const getDefaultFetchPolicy = (): FetchPolicy => 'network-only';
+export const getDefaultFetchPolicy = (): FetchPolicy => 'cache-and-network';
```

Comment:
Aaaand we're back! It has been `14 days` since the last Default Fetch Policy change. Do you happen to know if the [two issues](https://github.com/elastic/kibana/pull/33939#discussion_r269815362) @FrankHassanabad spoke of when it last changed are still an issue?

Reply:
Yes I put it back the way @FrankHassanabad wanted.